docs/user-guide/kubernetes.md

```diff
@@ -330,6 +330,72 @@ echo "$(minikube ip) traefik-ui.minikube" | sudo tee -a /etc/hosts
 
 We should now be able to visit [traefik-ui.minikube](http://traefik-ui.minikube) in the browser and view the Træfik Web UI.
 
+## Basic Authentication
+
+It's possible to add additional authentication annotations in the Ingress rule.
+The source of the authentication is a secret that contains usernames and passwords inside the the key auth.
```

Comment:
Please remove one of the `the` in `the the key auth.`

Reply:
good catch!

Reply:
@nmengin that typo appears to have been in the code base... It's live on the docs site.